src/include/migraphx/compile_options.hpp

```diff
@@ -32,9 +32,14 @@ inline namespace MIGRAPHX_INLINE_NS {
 
 struct compile_options
 {
-    bool offload_copy    = false;
+    /// Have MIGX allocate memory for the parameters and copy to and from the GPU
```

Comment:
This needs to be re-worded. 

It allocates memory for Parameters (need to check if it is pinned or not) and injects instructions that copies parameters and final output to and from GPU back to the Host.